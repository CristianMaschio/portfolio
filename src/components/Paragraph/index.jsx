import React, { PureComponent } from "react";
import PropTypes from "prop-types";

export default class Paragraph extends PureComponent {
  static propTypes = {
    id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
    index: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    image: PropTypes.string,
    sections: PropTypes.array.isRequired,
    moreSections: PropTypes.array,
  };

  state = {
    moreContent: false,
  };

  static defaultPropTypes = {
    index: 0,
    image: "",
    moreSections: [],
  };

  moreContent = React.createRef();

  //----------Functions-----------
  getSectionElements(sections) {
    return sections?.map((section, index) => {
      return (
        <div className="sectionContainer" key={index}>
          {section.img && (
            <div className="section-image">
              <img
                alt={`${section.title}`}
                src={require("../../assets/images/" + section.img)}
              />
            </div>
          )}
          <div className="sectionContent description">
            <p className="date">{section.date}</p>
            <h3>{section.title}</h3>
            {section.technologies ? (
              <div className="multy-section">
                <div className="sub-section">
                  <div className="sub-section-title">Technologies:</div>
                  <div
                    className="sub-section-content"
                    dangerouslySetInnerHTML={{ __html: section.technologies }}
                  />
                </div>
                <div className="sub-section">
                  <div className="sub-section-title">Achieving:</div>
                  <div
                    className="sub-section-content"
                    dangerouslySetInnerHTML={{ __html: section.achieving }}
                  />
                </div>
              </div>
            ) : (
              <div dangerouslySetInnerHTML={{ __html: section.description }} />
            )}
            {section.link && (
              <div className="link-container">
                <a
                  href={section.link}
                  target="_blank"
                  rel="noopener noreferrer"
                >
                  {"View"}
                </a>
              </div>
            )}
          </div>
        </div>
      );
    });
  }

  handleMoreContent = () => {
    const { moreContent } = this.state;
    if (!moreContent) {
      this.moreContent.current.style.height =
        this.moreContent.current.scrollHeight + 20 + "px";
    } else {
      this.moreContent.current.style.height = "0px";
    }

    this.setState({ moreContent: !moreContent });
  };

  //------------Render--------------
  render() {
    const { id, title, moreSections, sections } = this.props;
    const { moreContent } = this.state;
    return (
      <div id={id}>
        <div className="paragraph">
          <div className="paragraphContainer">
            <h2 className="paragraphTitle">{title}</h2>
          </div>
          <div className="paragraphContainer">
            <div className="sectionsContainer">
              {this.getSectionElements(sections)}
              <div ref={this.moreContent} className="more-container">
                {this.getSectionElements(moreSections)}
              </div>
            </div>
          </div>
        </div>
        {moreSections && moreSections.length >= 1 && (
          <p className="more-paragraph" onClick={this.handleMoreContent}>
            {!moreContent ? "more" : "less"}
          </p>
        )}
      </div>
    );
  }
}
