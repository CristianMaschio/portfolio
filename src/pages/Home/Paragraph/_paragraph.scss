.paragraph {
  margin-top: 2rem;

  .paragraphTitle {
    flex: 1;
    font-family: Roboto;
    font-weight: 300;
    font-size: 2.4rem;
    letter-spacing: -0.5px;
    color: $primary-color;
    mix-blend-mode: normal;
    text-align: center;
    @media screen and (max-width: $mobile-max-width) {
      font-size: 1.9rem;
    }
  }

  .leftTitle {
    text-align: left;
  }

  .rightTitle {
    text-align: right;
  }

  .paragraphContainer {
    display: flex;
    justify-content: center;
  }

  .paragraphImage {
    height: 5rem;
    align-self: center;
    @media screen and (max-width: $mobile-max-width) {
      height: 3rem;
    }
  }

  .sectionsContainer {
    flex: 1;
    align-self: center;
    font-family: Roboto;
    font-size: 0.8rem;
    letter-spacing: 0.5px;
    background-color: transparent;
    display: flex;
    flex-direction: column;
    align-items: center;
    margin: 0 0.5rem;
    .multy-section {
      .sub-section {
        display: flex;
        flex-direction: row;
        margin-top: 1rem;
        .sub-section-title {
          font-weight: bold;
          font-size: 0.8rem;
          min-width: 6rem;
        }
        .sub-section-content {
          text-align: left;
        }
      }
      @media screen and (max-width: $mobile-max-width) {
        .sub-section {
          display: flex;
          flex-direction: column;
          .sub-section-content {
            text-align: center;
          }
        }
      }
    }
  }

  .sectionContainer {
    display: flex;
    flex-direction: column;
    justify-content: center;
    max-width: 40rem;
    width: 100%;
    border-radius: 8px;
    margin: 1rem;
    box-shadow: 0 4px 8px 3px rgba(0, 0, 0, 0.1),
      0 6px 20px 0 rgba(0, 0, 0, 0.05);
    background-color: white;
    @media screen and (max-width: $mobile-max-width) {
      flex-wrap: wrap;
      margin: 1rem 0;
    }
  }

  .sectionContent {
    padding: 1rem;
    h3 {
      margin: 1.2rem 0 1rem 0;
      padding: 0;
      text-align: left;
      font-size: 1.2rem;
      @media screen and (max-width: $mobile-max-width) {
        text-align: center;
      }
    }
  }

  .section-image {
    display: flex;
    border-radius: 8px 8px 0 0;
    height: 3rem;
    background-color: rgba(59, 98, 196, 0.2);
    justify-content: center;
    img {
      width: auto;
      max-height: 2rem;
      margin: 0.5rem 0;
      border-radius: 4px;
    }
  }

  .date {
    min-width: 10rem;
    margin: 0;
    text-align: right;
    font-size: 0.8rem;
    height: 0px;
    @media screen and (max-width: $mobile-max-width) {
      text-align: center;
      padding-bottom: 0;
      margin-bottom: 0;
    }
  }

  .sectionContent.description {
    min-width: 25rem;
    font-size: 1rem;
    text-align: left;
    max-width: -webkit-fill-available;
    @media screen and (max-width: $mobile-max-width) {
      min-width: unset;
      width: 100%;
      text-align: center;
    }
  }

  .more-paragraph {
    text-align: center;
    cursor: pointer;
    color: $primary-color;
  }

  .link-container {
    display: flex;
    justify-content: flex-end;
    margin-top: 0.3rem;
  }
  a:link {
    color: $primary-color;
  }
}

.more-paragraph {
  text-align: center;
  cursor: pointer;
  color: $primary-color;
}

.more-container {
  height: 0px;
  overflow: hidden;
  width: 100%;
  transition: all 0.5s;
  display: flex;
  flex-direction: column;
  align-items: center;
}
