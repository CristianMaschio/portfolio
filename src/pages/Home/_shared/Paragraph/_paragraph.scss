.paragraph {
  margin: 1rem;
}

.leftLine {
  background-image: $primary-left-linear-gradient;
  height: 1px;
}

.rightLine {
  background-image: $primary-right-linear-gradient;
  height: 1px;
}

.paragraphTitle {
  flex: 1;
  font-family: Roboto;
  font-weight: 300;
  font-size: 1.9rem;
  letter-spacing: -0.5px;
  color: $primary-color;
  mix-blend-mode: normal;
  text-align: center;
}

.leftTitle {
  text-align: left;
}

.rightTitle {
  text-align: right;
}

.paragraphContainer {
  display: flex;
  justify-content: center;
}

.paragraphContainer.leftParagraph {
  flex-direction: row;
  flex-wrap: wrap-reverse;
}

.paragraphContainer.rightParagraph {
  flex-direction: row-reverse;
  flex-wrap: wrap-reverse;
}

.paragraphImage {
  height: 5rem;
  align-self: center;
  @media screen and (max-width: $mobile-max-width) {
    height: 3rem;
  }
}

.sectionsContainer {
  flex: 1;
  align-self: center;
  font-family: Roboto;
  font-size: 0.8rem;
  letter-spacing: 0.5px;
  background-color: #fff;
  z-index: 1;
  .multy-section {
    .sub-section {
      display: flex;
      flex-direction: row;
      margin: 0.5rem;
      .sub-section-title {
        font-weight: bold;
        font-size: 0.8rem;
        min-width: 6rem;
      }
      .sub-section-content {
        text-align: left;
      }
    }
    @media screen and (max-width: $mobile-max-width) {
      .sub-section {
        display: flex;
        flex-direction: column;
        margin: 0.5rem;
        .sub-section-title {
          font-weight: bold;
          font-size: 0.8rem;
          min-width: 6rem;
        }
        .sub-section-content {
          text-align: center;
        }
      }
    }
  }
}

.sectionContainer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  @media screen and (max-width: $mobile-max-width) {
    flex-wrap: wrap;
  }
}

.sectionContent {
  padding: 1rem;
  padding-top: 0px;
  max-width: 35rem;
  h3 {
    margin: 0;
    margin-bottom: 0.3rem;
    padding: 0;
    text-align: left;
    font-size: 1.2rem;
    @media screen and (max-width: $mobile-max-width) {
      text-align: center;
      margin: 0.3rem;
    }
  }
}

.sectionContent.date {
  min-width: 10rem;
  max-width: 12rem;
  text-align: center;

  @media screen and (max-width: $mobile-max-width) {
    padding-bottom: 0;
    margin-bottom: 0;
  }
}

.sectionContent.description {
  min-width: 25rem;
  font-size: 1rem;
  flex: 1;
  text-align: left;
  @media screen and (max-width: $mobile-max-width) {
    min-width: 100%;
    text-align: center;
  }
}

.more-paragraph {
  text-align: center;
  cursor: pointer;
  color: $primary-color;
  margin-top: -1rem;
}

.more-container {
  height: 0px;
  overflow: hidden;
  width: 100%;
  z-index: -1;
  transition: all 0.5s;
  .leftLine {
    margin-bottom: 0.7rem;
  }
  .rightLine {
    margin-bottom: 0.7rem;
  }
}
