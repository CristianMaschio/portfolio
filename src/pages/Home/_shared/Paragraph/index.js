import React, { PureComponent } from "react"
import PropTypes from "prop-types"

export default class Paragraph extends PureComponent {
  static propTypes = {
    id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
    index: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    image: PropTypes.string,
    sections: PropTypes.array.isRequired,
    moreSections: PropTypes.array,
  };

  static defaultPropTypes = {
    index: 0,
    image: "",
    moreSections: [],
  };

  state = {
    isLeft: undefined,
    moreContent: false,
    sectionsElements: undefined,
  };

  componentDidMount() {
    const { sections } = this.props

    //Create only one time all the sections
    const sectionsElements = this.getSectionElements(sections)

    this.setState({ isLeft: this.props.index % 2 === 0, sectionsElements })
  }

  moreContent = React.createRef();

  //----------Functions-----------

  //this fuction create all section components.
  getSectionElements(sections) {
    return (
      sections &&
      sections.map((section, index) => {
        return (
          <div className="sectionContainer" key={index}>
            <div className="sectionContent description">
              <p className="date">{section.date}</p>
              <h3>
                {section.title}{" "}
                {section.link && (
                  <a
                    href={section.link}
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    Link
                  </a>
                )}
              </h3>
              {section.technologies ? (
                <div className="multy-section">
                  <div className="sub-section">
                    <div className="sub-section-title">Technologies:</div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={{ __html: section.technologies }}
                    />
                  </div>
                  <div className="sub-section">
                    <div className="sub-section-title">Achieving:</div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={{ __html: section.achieving }}
                    />
                  </div>
                </div>
              ) : (
                <div
                  dangerouslySetInnerHTML={{ __html: section.description }}
                />
              )}
            </div>
          </div>
        )
      })
    )
  }

  handleMoreContent = () => {
    const { moreContent } = this.state
    if (!moreContent) {
      this.moreContent.current.style.height =
        this.moreContent.current.scrollHeight + "px"
    } else {
      this.moreContent.current.style.height = "0px"
    }

    this.setState({ moreContent: !moreContent })
  };

  //--------Render-Functions-------
  renderParagraphContent = () => {
    const { title, image, moreSections } = this.props
    const { isLeft, sectionsElements } = this.state

    return (
      <div className="paragraph">
        <div
          className={`paragraphContainer 
            ${
              isLeft ? "leftParagraph" : "rightParagraph"
            }
          `}
        >
          <img
            className="paragraphImage"
            alt="Universtity of Padua logo"
            src={image && require("../../../../assets/images/" + image)}
          />
          <h2 className="paragraphTitle">{title}</h2>
          <img
            className="paragraphImage"
            alt="Universtity of Padua logo"
            src={image && require("../../../../assets/images/" + image)}
          />
        </div>
        <div
          className={
            isLeft
              ? "paragraphContainer leftParagraph"
              : "paragraphContainer rightParagraph"
          }
        >
          <div className="sectionsContainer">
            {sectionsElements}
            <div ref={this.moreContent} className="more-container">
              {this.getSectionElements(moreSections)}
            </div>
          </div>
        </div>
      </div>
    )
  };

  //------------Render--------------

  render() {
    const { id, moreSections } = this.props
    const { isLeft, moreContent } = this.state
    return (
      <div id={id} style={{ paddingTop: "3rem" }}>
        <div className={isLeft ? "leftLine" : "rightLine"} />
        {this.renderParagraphContent()}
        {moreSections && moreSections.length >= 1 ? (
          <p className="more-paragraph" onClick={this.handleMoreContent}>
            {!moreContent ? "more" : "less"}
          </p>
        ) : (
          ""
        )}
      </div>
    )
  }
}
