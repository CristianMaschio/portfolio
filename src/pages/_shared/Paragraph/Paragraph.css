.paragraph {
  margin: 1rem;
}

.leftLine {
  background-image: linear-gradient(to left, #3b62b1, transparent);
  height: 1px;
}

.rightLine {
  background-image: linear-gradient(to right, #3b62b1, transparent);
  height: 1px;
}

.paragraphTitle {
  font-family: Roboto;
  font-weight: 200;
  font-size: 2rem;
  letter-spacing: -0.5px;
  color: #3b62b1;

  mix-blend-mode: normal;
}

.leftTitle {
  text-align: left;
}

.rightTitle {
  text-align: right;
}

.paragraphContainer {
  display: flex;
  height: 200px;
  justify-content: center;
}

.paragraphContainer.leftParagraph {
  flex-direction: row;
  flex-wrap: wrap-reverse;
}

.paragraphContainer.rightParagraph {
  flex-direction: row-reverse;
  flex-wrap: wrap-reverse;
}

.paragraphImage {
  height: 150px;
  width: 150px;
  align-self: center;
}

.sectionsContainer {
  flex: 1;
  align-self: center;
  font-family: Roboto;
  font-size: 0.8rem;
  letter-spacing: 0.5px;
}

.sectionContainer {
  display: flex;
  flex-direction: row;
}

.sectionContent {
  padding: 1rem;
  padding-top: 0px;
}

.sectionContent.date {
  min-width: 10rem;
}

.sectionContent.description {
  min-width: 25rem;
}
