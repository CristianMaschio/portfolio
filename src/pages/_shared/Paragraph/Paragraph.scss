@import "../../../styles/colors";

.paragraph {
  margin: 1rem;
}

.leftLine {
  background-image: $primary-left-linear-gradient;
  height: 1px;
}

.rightLine {
  background-image: $primary-right-linear-gradient;
  height: 1px;
}

.paragraphTitle {
  flex: 1;
  font-family: Roboto;
  font-weight: 200;
  font-size: 2rem;
  letter-spacing: -0.5px;
  color: $primary-color;
  mix-blend-mode: normal;
  text-align: center;
}

.leftTitle {
  text-align: left;
}

.rightTitle {
  text-align: right;
}

.paragraphContainer {
  display: flex;
  justify-content: center;
}

.paragraphContainer.leftParagraph {
  flex-direction: row;
  flex-wrap: wrap-reverse;
}

.paragraphContainer.rightParagraph {
  flex-direction: row-reverse;
  flex-wrap: wrap-reverse;
}

.paragraphImage {
  height: 5rem;
  align-self: center;
  @media screen and (max-width: $mobile-max-width) {
    height: 3rem;
  }
}

.sectionsContainer {
  flex: 1;
  align-self: center;
  font-family: Roboto;
  font-size: 0.8rem;
  letter-spacing: 0.5px;
}

.sectionContainer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  @media screen and (max-width: $mobile-max-width) {
    flex-wrap: wrap;
  }
}

.sectionContent {
  padding: 1rem;
  padding-top: 0px;
  max-width: 35rem;
  h3 {
    margin: 0;
    margin-bottom: 0.3rem;
    padding: 0;
    text-align: left;
    @media screen and (max-width: $mobile-max-width) {
      text-align: center;
      margin: 0.3rem;
    }
  }
}

.sectionContent.date {
  min-width: 10rem;
  max-width: 12rem;
  align-self: center;
  text-align: center;

  @media screen and (max-width: $mobile-max-width) {
    padding-bottom: 0;
    margin-bottom: 0;
  }
}

.sectionContent.description {
  min-width: 25rem;
  flex: 1;
  text-align: left;
  @media screen and (max-width: $mobile-max-width) {
    min-width: 100%;
    text-align: center;
  }
}

.more-paragraph {
  text-align: center;
  cursor: pointer;
}
