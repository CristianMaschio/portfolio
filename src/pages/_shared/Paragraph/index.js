import React, { PureComponent } from "react"
import "./Paragraph.css"

export default class Paragraph extends PureComponent {
  state = {
    isLeft: undefined
  }

  componentDidMount(){
    this.setState({isLeft: this.props.index % 2 === 0})
  }

  //----------Functions-----------

  renderParagraphContent = () => {
    const { title, image, sections } = this.props
    const isLeft = this.state.isLeft

    return(
      <div className="paragraph">
        <h2
          className={
            isLeft ? "paragraphTitle leftTitle" : "paragraphTitle rightTitle"
          }
        >
          {title}
        </h2>
        <div
          className={
            isLeft
              ? "paragraphContainer leftParagraph"
              : "paragraphContainer rightParagraph"
          }
        >
          <div className="sectionsContainer">
            {sections.map((section, index) => {
              return (
                <div key={index}>
                  <p>{section.date}</p>
                </div>
              )
            })}
          </div>
          <img
            className="paragraphImage"
            alt="Universtity of Padua logo"
            src={image && require("../../../../public/images/" + image)}
          />
        </div>
      </div>
    )
  }

  //------------Render--------------

  render() {
    const isLeft = this.state.isLeft
    return (
      <>
        <div className={isLeft ? "leftLine" : "rightLine"} />
        {this.renderParagraphContent()}
      </>
    )
  }
}
