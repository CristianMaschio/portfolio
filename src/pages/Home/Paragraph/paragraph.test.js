import React from "react"
import renderer from "react-test-renderer"
import Paragraph from "."

it("Paragraph: render correctly", () => {
  const paragraph = renderer
    .create(
      <Paragraph
        id="test"
        index={0}
        title={"test"}
        sections={[
          {
            img: "gestionaleauto.png",
            date: "05/2019 - 30/09/2019",
            title: "GestionaleAuto.com - React Developer",
            technologies:
              "React, Javascript, Redux, Redux-Form, Redux-Saga, Html & Scss.",
            achieving:
              "Learned to working in Agile Scrum team: my daily tasks ranged from refactoring to develop new feature for a large project code base of a management software. In which I managed to improve the simplicity and performance of some components, making them 30-40% faster.",
            link: "https://gestionaleauto.com/",
          },
          {
            img: "discontinuity.png",
            date: "10/2018 - 04/2019",
            title: "Discontinuity - Full Stack Developer",
            technologies:
              "React Native, React, Javascript, Gatsby, AWS, DynamoDB, GraphQL, Html & Scss.",
            achieving:
              "Working in Agile team, with maintaining and implementing applications and web sites from scratch",
            link: "http://discontinuity.eu/",
          },
        ]}
      />
    )
    .toJSON()

  expect(paragraph).toMatchSnapshot()
})
 