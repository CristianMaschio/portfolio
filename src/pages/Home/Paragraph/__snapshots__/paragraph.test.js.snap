// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph: render correctly 1`] = `
<div
  id="test"
>
  <div
    className="paragraph"
  >
    <div
      className="rightLine"
    />
    <div
      className="paragraphContainer"
    >
      <img
        alt="Universtity of Padua logo"
        className="paragraphImage"
      />
      <h2
        className="paragraphTitle"
      >
        test
      </h2>
      <img
        alt="Universtity of Padua logo"
        className="paragraphImage"
      />
    </div>
    <div
      className="paragraphContainer"
    >
      <div
        className="sectionsContainer"
      >
        <div
          className="sectionContainer"
        >
          <div
            className="sectionContent description"
          >
            <div
              className="section-image"
            >
              <img
                alt="GestionaleAuto.com - React Developer"
                src="gestionaleauto.png"
              />
            </div>
            <p
              className="date"
            >
              05/2019 - 30/09/2019
            </p>
            <h3>
              GestionaleAuto.com - React Developer
            </h3>
            <div
              className="multy-section"
            >
              <div
                className="sub-section"
              >
                <div
                  className="sub-section-title"
                >
                  Technologies:
                </div>
                <div
                  className="sub-section-content"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "React, Javascript, Redux, Redux-Form, Redux-Saga, Html & Scss.",
                    }
                  }
                />
              </div>
              <div
                className="sub-section"
              >
                <div
                  className="sub-section-title"
                >
                  Achieving:
                </div>
                <div
                  className="sub-section-content"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "Learned to working in Agile Scrum team: my daily tasks ranged from refactoring to develop new feature for a large project code base of a management software. In which I managed to improve the simplicity and performance of some components, making them 30-40% faster.",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="link-container"
            >
              <a
                href="https://gestionaleauto.com/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Go to page &gt;&gt;
              </a>
            </div>
          </div>
        </div>
        <div
          className="sectionContainer"
        >
          <div
            className="sectionContent description"
          >
            <div
              className="section-image"
            >
              <img
                alt="Discontinuity - Full Stack Developer"
                src="discontinuity.png"
              />
            </div>
            <p
              className="date"
            >
              10/2018 - 04/2019
            </p>
            <h3>
              Discontinuity - Full Stack Developer
            </h3>
            <div
              className="multy-section"
            >
              <div
                className="sub-section"
              >
                <div
                  className="sub-section-title"
                >
                  Technologies:
                </div>
                <div
                  className="sub-section-content"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "React Native, React, Javascript, Gatsby, AWS, DynamoDB, GraphQL, Html & Scss.",
                    }
                  }
                />
              </div>
              <div
                className="sub-section"
              >
                <div
                  className="sub-section-title"
                >
                  Achieving:
                </div>
                <div
                  className="sub-section-content"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "Working in Agile team, with maintaining and implementing applications and web sites from scratch",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="link-container"
            >
              <a
                href="http://discontinuity.eu/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Go to page &gt;&gt;
              </a>
            </div>
          </div>
        </div>
        <div
          className="more-container"
        />
      </div>
    </div>
  </div>
  
</div>
`;
